src/test_l1_dsAudio.c

```diff
@@ -10102,10 +8011,6 @@ int test_l1_dsAudio_register ( void )
 	UT_add_test( pSuite, "dsAudioPortTerm_L1_negative" ,test_l1_dsAudio_negative_dsAudioPortTerm );
 	UT_add_test( pSuite, "dsGetAudioPort_L1_positive" ,test_l1_dsAudio_positive_dsGetAudioPort );
 	UT_add_test( pSuite, "dsGetAudioPort_L1_negative" ,test_l1_dsAudio_negative_dsGetAudioPort );
```

Comment:
test cases for below APIs are missing:

dsAudioEnableARC
dsAudioFormatUpdateRegisterCB
dsAudioSetSAD
dsGetSupportedARCTypes

Reply:
Done

Reply:
hi bhanu,
While developing L1 test cases we agreed to move the test cases to L2 which has dependency of external devices.
Do we need to add it again ?